### ISSUE TITLE
_this2.builder.destroy() is not a function

### ISSUE THREAD

**Issue by shamanking** [formio/react-formio#115]
I use FormBuilder, as:
```
render() {
      return (
        <div id="content" className="app-content" role="main">
          <div className="wrapper-md">
            <FormBuilder
              form={{ display: 'form' }}
              onChange={(schema) => this.onChangeFormBuilder(schema)}
            />
          </div>
        </div>
      );
    }
```
When I move to another component, it has error:

![screen shot 2018-05-18 at 10 48 13 am](https://user-images.githubusercontent.com/5752503/40215193-562c030c-5a89-11e8-8a73-24ef09ad1b07.png)

How can I fixed it ?

**Comment by Dmitrylolo** [comment]
I think, it was in advanced react native lessons by Stephen Grider. Video number 137 -callback-gotcha. When you using a callback from child component, you have to bind it in parent component

**Comment by rmillener** [comment]
We're also having this issue when routing to a different page using v2.1.0

@randallknutson are we missing something?

**Comment by randallknutson** [comment]
Hi all,

Thanks for the bug report. This should be fixed in version 2.1.1
